docker/pypi/wmagent/README.md

```diff
@@ -382,7 +382,7 @@ They are set at different moments along the process: i.e. at `buildtime` - when
    * RUCIO_ACCOUNT=wma_test
    * RUCIO_HOST=http://cms-rucio-int.cern.ch
    * RUCIO_AUTH=https://cms-rucio-auth-int.cern.ch
-
+   * MSPILEUP_URL=https://***.cern.ch/ms-pileup/data/pileup
```

Comment:
Are the three *** in the actual url correct? @amaltaro

Reply:
It's just to hide the actual endpoint, this way it's one endpoint less for us to remove from GitHub for future us.

Reply:
it would have been good to at least start it with the `https://cmsweb***.cern.ch.....`